politeiawww/invoices.go

```diff
@@ -442,6 +442,17 @@ func validateInvoice(ni cms.NewInvoice, u *user.User) error {
 				}
 			}
 
+			// Validate month/year to make sure the first day of the following
+			// month is after the current date.  For example, if a user submits
+			// an invoice for 03/2019, the first time that they could submit an
+			// invoice would be approx. 12:01 AM 04/01/2019
+			startOfFollowingMonth := time.Date(int(invInput.Year), time.Month(invInput.Month+1), 0, 0, 0, 0, 0, time.UTC)
+			if startOfFollowingMonth.After(time.Now()) {
```

Comment:
Shouldn't this be `startOfFollowingMonth.Before(time.Now())`, since if `startOfFollowingMonth` represents "4/1/19 00:00:00", the current time must be any time after that.

Reply:
So this check errors if the 4/1/19 00:00:00 is AFTER the current time (let's say, 3/30/19 00:00:00).  I believe it's correct since we only want people to be able to submit invoices when the date is (not) After the end of the invoice period.  Does that make sense?

Reply:
Oh, duh... my bad